LeBlue/src/main/java/houtbecke/rs/le/fourdotthree/LeRemoteDevice43.java

```diff
@@ -89,7 +89,10 @@ public void close() {
 
         }
     }
-
+    @Override
+    public void startServicesDiscovery(UUID... uuids){
+        this.startServicesDiscovery();
```

Comment:
Could you add a comment here to say this is not done on Android because it is known to be buggy?